packages/core/src/utils/TypeUtil.ts

```diff
@@ -103,6 +103,7 @@ export interface WcWallet {
   image_url?: string
   order?: number
   mobile_link?: string | null
+  link_mode?: string | null
```

Comment:
this might bring confusion as we are developing another feature called link-mode 👀

Reply:
this comes from Cloud https://github.com/WalletConnect/cloud-app/pull/1195